mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1580,6 +1580,24 @@ func ResourceContainerCluster() *schema.Resource {
 				},
 			},
 
+			"additional_pod_ranges_config": {
```

Comment:
In the API doc, `additionalPodRangesConfig` is inside `IPAllocationPolicy`. Terraform provider will follow the API doc. Can the field `additional_pod_ranges_config` be moved inside `ip_allocation_policy` ?

https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#ipallocationpolicy

Reply:
Done.